yaml/pois.yaml

```diff
@@ -414,3 +414,45 @@ filters:
     min_zoom: 15
     output:
       kind: {col: tags->whitewater}
+  - filter: {aerialway: pylon}
```

Comment:
Let's split off discussion of where the new power_pole and power_tower should go into https://github.com/mapzen/vector-datasource/issues/715 and move forward with merging this PR.